flood_adapt/config.py

```diff
@@ -0,0 +1,63 @@
+import os
+from pathlib import Path
+
+import tomli
+
+
+def set_database_root(database_root):
+    """
+    Sets the root directory for the database.
+
+    Args:
+        database_root (str): The new root directory path.
+    """
+    os.environ["DATABASE_ROOT"] = database_root
+
+
+def set_system_folder(system_folder):
+    """
+    Sets the system folder path.
+
+    Args:
+        system_folder (str): The new system folder path.
+    """
+    os.environ["SYSTEM_FOLDER"] = system_folder
+
+
+def set_site_name(site_name):
+    """
+    Sets the site_name.
+
+    Args:
+        system_folder (str): The new system folder path.
+    """
+    os.environ["SITE_NAME"] = site_name
```

Comment:
Add a check verifying whether the specified path exists.
If not, `raise` an exception.